src/DeviceInput/Helpers/eventFactory.ts

```diff
@@ -50,6 +50,8 @@ export class DeviceEventFactory {
 
         evt.pointerId = deviceType === DeviceType.Mouse ? 1 : deviceSlot;
 
+        evt.pointerType = deviceType === DeviceType.Mouse ? "mouse" : "touch";

```

Comment:
nit: can we enclose in a if with the previous line ?

Reply:
Fixed